<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="5" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-17T01:00:31 EET" name="All Test Suite" finished-at="2022-09-17T01:00:47 EET" duration-ms="16757">
    <groups>
    </groups>
    <test started-at="2022-09-17T01:00:31 EET" name="E:/ITworx/src/test" finished-at="2022-09-17T01:00:47 EET" duration-ms="16757">
      <class name="tests.AuthTest">
        <test-method is-config="true" signature="setEnvironment()[pri:0, instance:tests.AuthTest@43d455c9]" started-at="2022-09-17T01:00:25 EET" name="setEnvironment" finished-at="2022-09-17T01:00:31 EET" duration-ms="5911" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setEnvironment -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AuthTest@43d455c9]" started-at="2022-09-17T01:00:31 EET" name="setUp" finished-at="2022-09-17T01:00:31 EET" duration-ms="24" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="RegisterTest()[pri:0, instance:tests.AuthTest@43d455c9]" started-at="2022-09-17T01:00:31 EET" name="RegisterTest" finished-at="2022-09-17T01:00:41 EET" duration-ms="9972" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RegisterTest -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.AuthTest@43d455c9]" started-at="2022-09-17T01:00:41 EET" name="setUp" finished-at="2022-09-17T01:00:41 EET" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="login()[pri:0, instance:tests.AuthTest@43d455c9]" started-at="2022-09-17T01:00:41 EET" name="login" finished-at="2022-09-17T01:00:43 EET" duration-ms="2331" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AuthTest@43d455c9]" started-at="2022-09-17T01:00:47 EET" name="tearDown" finished-at="2022-09-17T01:00:47 EET" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.AuthTest -->
      <class name="tests.PurchaseItem">
        <test-method is-config="true" signature="setUp()[pri:0, instance:tests.PurchaseItem@6d868997]" started-at="2022-09-17T01:00:43 EET" name="setUp" finished-at="2022-09-17T01:00:43 EET" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="purchasingProduct()[pri:0, instance:tests.PurchaseItem@6d868997]" started-at="2022-09-17T01:00:43 EET" name="purchasingProduct" finished-at="2022-09-17T01:00:47 EET" duration-ms="4011" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:208)
at org.openqa.selenium.interactions.Actions.<init>(Actions.java:65)
at Pages.PageBase.hoverAndClickOnElement(PageBase.java:47)
at Pages.HomePage.openWomenList(HomePage.java:31)
at tests.PurchaseItem.purchasingProduct(PurchaseItem.java:9)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- purchasingProduct -->
      </class> <!-- tests.PurchaseItem -->
    </test> <!-- E:/ITworx/src/test -->
  </suite> <!-- All Test Suite -->
</testng-results>
